01_getting_started/hello_world.py

```diff
@@ -15,20 +15,17 @@
 
 import modal
 
-app = modal.App(
-    "example-hello-world"
-)  # Note: prior to April 2024, "app" was called "stub"
+app = modal.App("example-hello-world")
```

Comment:
I was initially resistant to removing this comment -- what if new users shortly encounter `Stub` and get confused -- but on further reflection I think we should just remove it now. There are other places where they can learn that information, and the hello world should be clean.